presentation-api/controlling-ua/PresentationConnection_onmessage-manual.html

```diff
@@ -38,6 +38,7 @@
 
     presentBtn.onclick = () => {
         presentBtn.disabled = true;
+        const stash = new Stash('9bf08fea-a71a-42f9-b3c4-fa19499e4d12', 'f1fdfd10-b606-4748-a644-0a8e9df3bdd6');
```

Comment:
With the risk of contradicting myself once more... Not sure how these IDs will evolve over time, but as opposed to the message variables which mostly do not have to be the same on the controlling side and on the receiving side (well, except message5 for the "send" test), the IDs need to be the same on both sides, so I think it would be valuable to define them in some common place, e.g. in `common.js`.

Also, naming them would help understand that one ID is used for the controller-to-receiver direction and one for the receiver-to-controller direction.